lib/src/Channel.cpp

```diff
@@ -4,9 +4,12 @@
 
 using namespace sairedis;
 
+#define REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS (60*1000)
```

Comment:
I would like to use this define in sonic-swss/orchagent/saihelper.cpp
Can you please move this define to .h file that can be included from both Channel.cpp and saihelper.cpp?
If there is such...

Reply:
why would you need this define on saihelper?

Reply:
I am using the new API to change the timeout and later on I want to change it back to default, this is where I want to use your define.
In the PR bellow I used new define SAI_REDIS_GET_RESPONSE_TIMEOUT_MS_DEFAULT but I would like to replace it to REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS if it is possible.
[(https://github.com/liorghub/sonic-swss/pull/3)]

Reply:
updated

Reply:
i was actually thinking, that this define maybe should not be exposed as define here, but you should do SAI api GET api, to obtain this attribute default value

Reply:
Generally, if we have set API we can have get API, basic stuff.
However, there is nothing wrong with exposing the default value in header file and for my needs it will do the work just fine. I usually vote for the option with less lines of code.

Reply:
ok i leave it as is